main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import "fmt"
+
+func average(numbers []int) float64 {
+	sum := 0
+	for _, n := range numbers {
+		sum += n
+	}
+	return float64(sum) / float64(len(numbers))
```

Comment:
What happens if len(numbers) == 0 ?
Please add a check that slice length is greater than 0.